shared/config.json

```diff
@@ -60,5 +61,21 @@
                 "propagate": true
             }
         }
+    },
+    "_frontend": "Configuration used by the frontend.",
```

Comment:
still feel like this could maybe be nested or something, but maybe that complicates things too much

Reply:
Yeah the aim is to be able to share some of the configs. Nesting makes that difficult...